readthedocs/core/permissions.py

```diff
@@ -74,6 +74,13 @@ def projects(cls, user, admin=False, member=False):
 
         return projects
 
+    @classmethod
+    def organizations(cls, user, admin=False, member=False):
+        from readthedocs.organizations.models import Organization
+
+        projects = Subquery(cls.projects(user, admin, member).values("id"))
```

Comment:
Same here, you don't need a subquery.

Reply:
Also, this is returning projects where the user is admin, and then returning the related organizations, this isn't the same as returning the organizations where the user is owner (admin). We already have a `members()`, `admins()` and `owners()` methods.

Reply:
Good point! Here I need "all the organizations where the user is owner" called in this way `AdminPermission.organizations(user, owner=True, member=False)`